src/keys.c

```diff
@@ -371,7 +373,10 @@ load_keys(const char* jwkdir)
             filepath[sizeof(filepath) - 1] = '\0';
             json_auto_t* json = json_load_file(filepath, 0, NULL);
```

Comment:
Use the `error` here, i.e. `json_load_file(filepath, 0, &error);`

Reply:
Doh, copied the working patch and missed that.